terraform/modules/vault/providers.tf

```diff
@@ -2,6 +2,10 @@
 #
 # Copyright (c) 2014-2019 Avant, Sean Lingren
 
+provider "aws" {
```

Comment:
This shouldn't be necessary, did you have an issue that required you to add it? Technically we shouldn't need the `dr` provider either but we've kept it in to make it easier on the user. Adding providers in the module (instead of inheriting them from the module caller) can cause issues when deleting the module.

Some docs:
https://www.terraform.io/docs/configuration/modules.html#providers-within-modules

Reply:
I had an issue w/o the provided defined, but I cannot recall which error exactly. I'll remove it. Unless someone else complains, it may be configuration-related